service/src/main/java/uk/gov/hmcts/reform/fpl/enums/Cardinality.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+public enum Cardinality {
+    ZERO, ONE, MANY;
+
+    public static Cardinality from(int amount) {
+
+        if (amount < 0) {
+            throw new IllegalArgumentException("Cardinality can be represented by non negative numbers only");
```

Comment:
This is an incredibly pedantic suggestion but rather than "non negative numbers only", could we use "non negative whole numbers only"?